TrashMob/client-app/src/components/Pages/Home.tsx

```diff
@@ -53,100 +53,138 @@ const Home: FC<HomeProps> = ({ isUserLoaded, currentUser, history, location, mat
-                            <img src={Logo} alt='TrashMob.eco logo' className='banner-logo' />
-                            <h3 className='ml-md-4 mt-4 mb-4 mb-md-5 font-weight-bold font-size-xl banner-heading pl-3'>
-                                Meet up. Clean up. Feel good.
-                            </h3>
-                            <Link className='btn btn-primary ml-5 py-md-3 banner-button' to='/gettingstarted'>
-                                Join us today
-                            </Link>
-                            <Button
-                                className='btn btn-primary ml-5 py-md-3 banner-button'
-                                onClick={() => handleShowModal(true)}
-                            >
-                                <Share className='mr-2' />
-                                Invite a friend
-                            </Button>
+                    <div className='d-flex flex-column px-0 py-0 pl-lg-5 m-auto w-100' style={{ zIndex: 10 }}>
+                        <div className='d-flex flex-row-reverse align-content-center mx-0 px-0'>
+                            <img
+                                src={Globe2}
+                                className='h-100 w-auto d-none d-sm-block mt-0'
+                                alt='Globe'
+                                style={{
+                                    maxHeight: '300px',
+                                    maxWidth: '300px',
+                                }}
+                            />
+                            <img
+                                src={Globe2}
+                                className='d-block d-sm-none h-100 w-auto m-auto position-absolute'
+                                alt='Globe'
+                                style={{
+                                    opacity: '30%',
+                                    zIndex: '-1',
+                                }}
+                            />
+                            <div className='d-flex flex-column justify-content-center w-75 m-auto z-10 pl-sm-2 px-md-5'>
+                                <img
+                                    src={Logo}
+                                    className='w-100 mx-auto mt-4 ml-sm-3'
+                                    alt='Trashmob.eco logo'
+                                    style={{ maxWidth: '550px' }}
+                                />
+                                <h4 className='text-center text-sm-left mb-4 ml-sm-5 font-weight-bold banner-heading ml-sm-4'>
+                                    Meet up. Clean up. Feel good.
+                                </h4>
+                                <Link
+                                    className='mb-3 mb-sm-4 btn btn-primary mx-auto ml-sm-5 banner-button mr-sm-auto ml-sm-4'
+                                    to='/gettingstarted'
+                                >
+                                    Join us today
+                                </Link>
```

Comment:
The "Invite a friend" Button is missing, is that intentional?

Reply:
It's possible I introduced the issue, the conflicts resolution between the changes that were made on the homepage and the responsiveness were not trivial.flicts resolution between the changes that were made on the homepage and the responsiveness were not trivial. I`m npt sure maybe it`s intentional.

Reply:
I took it off in my original push to redo to responsiveness because I was following the design page too closely. It started getting out of scope, so I undid the "new design implementations" I had started, but missed resetting this one. Good catch!